src/main/java/seedu/fridgefriend/command/SearchCommand.java

```diff
@@ -54,10 +56,10 @@ public String getMessagePrintedToUser() {
         if (!isContain) {
             LoggingHandler.logInfo("Search for food unsuccessful: No " + foodName + " found.");
             return "You do not have " + foodName + " in your fridge.";
+        } else {
+            LoggingHandler.logInfo("Search for food successful: " + foodName + " found.");
+            return message.toString();
```

Comment:
Would it be better if you refactored/SLAPPed the Search function more? Right now, "getMessagePrintedToUser" looks quite messy. It might be a good idea to extract out into several methods for SLAP :-)